x-pack/solutions/search/plugins/enterprise_search/server/deprecations/index.ts

```diff
@@ -59,11 +61,18 @@ export const getRegisteredDeprecations = (
         true // only counts if the Fleet Server is active
       );
       const entSearchDetails = getEnterpriseSearchNodeDeprecation(config, cloud, docsUrl);
-      const [crawlerDetails, nativeConnectorsDetails] = await Promise.all([
-        await getCrawlerDeprecations(ctx, docsUrl),
-        await getNativeConnectorDeprecations(ctx, hasAgentless, hasFleetServer, cloud, docsUrl),
-      ]);
-      return [...entSearchDetails, ...crawlerDetails, ...nativeConnectorsDetails];
+      const [crawlerDetails, nativeConnectorsDetails, entSearchIndexIncompatibility] =
+        await Promise.all([
+          await getCrawlerDeprecations(ctx, docsUrl),
+          await getNativeConnectorDeprecations(ctx, hasAgentless, hasFleetServer, cloud, docsUrl),
+          await getEnterpriseSearchPre8IndexDeprecations(ctx, docsUrl, config.host),
+        ]);
```

Comment:
sorry I missed this in the original PR but these inner `awaits` should not be here.

```suggestion
        await Promise.all([
          getCrawlerDeprecations(ctx, docsUrl),
          getNativeConnectorDeprecations(ctx, hasAgentless, hasFleetServer, cloud, docsUrl),
          getEnterpriseSearchPre8IndexDeprecations(ctx, docsUrl, config.host),
        ]);
```